src/commands/synthetics/mobile.ts

```diff
@@ -26,6 +30,14 @@ export const uploadMobileApplications = async (
   )
 
   const fileBuffer = await fs.promises.readFile(applicationPathToUpload)
+  const fileBufferBytesSize = fileBuffer.byteLength
+  if (fileBufferBytesSize < MIN_APPLICATION_SIZE || MAX_APPLICATION_SIZE < fileBufferBytesSize) {
+    throw new CriticalError(
+      'INVALID_MOBILE_APPLICATION_SIZE',
+      `Invalid Mobile Application size. Expect a size between ${MIN_APPLICATION_SIZE}B and ${MAX_APPLICATION_SIZE}B, got ${fileBufferBytesSize}B.`
```

Comment:
Is it really complicated to format in a human readable way the size displayed ? To avoid the `1073741824B`